.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
```

Comment:
Also, we need to specify explicitly somehow, which version of symengine is required. I think conda allows that, doesn't it? So if we release symengine, can we use the git tag to reference a conda package?